es/news/_posts/2016-12-25-ruby-2-4-0-released.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: news_post
+title: "Liberado Ruby 2.4.0"
```

Comment:
However, "Liberado Ruby ..." has previously been used **very** often, too. And it has the "full name" of the version (i.e. "Ruby 2.4.0") in unmodified form.

But you must decide what sounds better in `es`.